lib/RoleDetails/RoleDetailsAssignUsers/RoleDetailsAssignUsers.js

```diff
@@ -35,6 +38,22 @@ export const RoleDetailsAssignUsers = ({
   const { mutateDeleteUserRoles } = useDeleteUserRolesMutation(sendErrorCallout, { tenantId });
   const initialSelectedUsers = useMemo(() => keyBy(selectedUsers, 'id'), [selectedUsers]);
 
+  const handleCheckKeycloakUsers = async (newSelectedUsers) => {
+    const userIdsSet = new Set([...newSelectedUsers.map(x => x.id), ...Object.values(initialSelectedUsers).map(x => x.id)]);
+    const keycloakNoneExistingUsersList = await checkAndGetNoneKeycloakAuthUsers({
```

Comment:
I think we want the variables to be named like this: 
```suggestion
    const nonKeycloakUsersList = await getNonKeycloakUsers({
```